gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Job.java

```diff
@@ -53,15 +123,7 @@ public Job(BigQuery bigquery, JobInfo info) {
    * @throws BigQueryException upon failure
    */
   public static Job get(BigQuery bigquery, String job, BigQuery.JobOption... options) {
```

Comment:
Do you think we can remove static methods like this one from functional classes?  Before this PR it made sense, because the service object didn't return funcitonal objects.  Now that the service object's `get` method returns a functional object, these methods are obsolete.